eng/MSBuild/ProjectStaging.targets

```
@@ -28,4 +28,10 @@
     <Description Condition="'$(Stage)' == 'obsolete'">Obsolete Package. $(Description)</Description>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <SuppressFinalPackageVersion>false</SuppressFinalPackageVersion>
```

NIT: Not super important, but in arcade the default is actually to have unset, rather than set to false. Doesn't matter unless there are checks that do `SuppressFinalPackageVersion` != ``
```suggestion
    <SuppressFinalPackageVersion />
```